### ISSUE TITLE
lora 微调 Ovis2-34B loss=0.0

### ISSUE THREAD

**Issue by YUANMU227** [modelscope/ms-swift#4711]
微调Ovis2-34B，--loss=0.0 grad_norm=nan
微调Ovis2-34B，从第一个step开始就有这个问题
ms_swift==3.5.2，transformers==4.46.2
升级到最新版本还是有这个问题：ms_swift==3.6.0.dev0，transformers==4.52.4

> {"loss": 0.0, "grad_norm": 0.0, "learning_rate": 1.538e-05, "memory(GiB)": 72.64, "train_speed(iter/s)": 0.01084, "epoch": 0.00710637, "global_step/max_steps": "1/420", "percentage": "0.24%", "elapsed_time": "1m 30s", "remaining_time": "10h 30m 51s"}
> {"loss": 0.0, "grad_norm": 0.0, "learning_rate": 3.077e-05, "memory(GiB)": 72.64, "train_speed(iter/s)": 0.011111, "epoch": 0.01421275, "global_step/max_steps": "2/420", "percentage": "0.48%", "elapsed_time": "2m 58s", "remaining_time": "10h 20m 21s"}
> {"loss": 0.0, "grad_norm": 0.0, "learning_rate": 4.615e-05, "memory(GiB)": 72.64, "train_speed(iter/s)": 0.011198, "epoch": 0.02131912, "global_step/max_steps": "3/420", "percentage": "0.71%", "elapsed_time": "4m 25s", "remaining_time": "10h 16m 13s"}
> {"loss": 0.0, "grad_norm": 0.0, "learning_rate": 6.154e-05, "memory(GiB)": 72.64, "train_speed(iter/s)": 0.011245, "epoch": 0.02842549, "global_step/max_steps": "4/420", "percentage": "0.95%", "elapsed_time": "5m 53s", "remaining_time": "10h 13m 15s"}
> {"loss": 0.0, "grad_norm": 0.0, "learning_rate": 7.692e-05, "memory(GiB)": 72.64, "train_speed(iter/s)": 0.011278, "epoch": 0.03553187, "global_step/max_steps": "5/420", "percentage": "1.19%", "elapsed_time": "7m 21s", "remaining_time": "10h 10m 38s"}
> {"loss": 0.0, "grad_norm": 0.0, "learning_rate": 9.231e-05, "memory(GiB)": 72.64, "train_speed(iter/s)": 0.011301, "epoch": 0.04263824, "global_step/max_steps": "6/420", "percentage": "1.43%", "elapsed_time": "8m 48s", "remaining_time": "10h 8m 20s"}

使用的shell如下：

> CUDA_VISIBLE_DEVICES=0,1 MASTER_PORT=29501 NPROC_PER_NODE=2 swift sft \
>     --model_type ovis2 \
>     --model ./Ovis2-34B \
>     --train_type lora \
>     --dataset ./train.jsonl \
>     --val_dataset ./test.jsonl \
>     --torch_dtype bfloat16 \
>     --deepspeed ./scripts/zero2.json \
>     --num_train_epochs 3 \
>     --per_device_train_batch_size 2 \
>     --per_device_eval_batch_size 1 \
>     --gradient_accumulation_steps 16 \
>     --eval_steps 10 \
>     --save_steps 10 \
>     --save_total_limit 70 \
>     --learning_rate 2e-4 \
>     --lora_rank 8 \
>     --lora_alpha 32 \
>     --target_modules all-linear \
>     --logging_steps 1 \
>     --max_length 2048 \
>     --output_dir ./results/Ovis2-34B_train \
>     --system 'You are a helpful assistant.' \
>     --warmup_ratio 0.03 \
>     --dataloader_num_workers 4 \
>     --model_author swift \
>     --model_name swift-robot

辛苦看下这个问题，感谢

相关问题：
https://github.com/modelscope/ms-swift/issues/3494

**Comment by Jintao-Huang** [comment]
--max_length 2048 

这个调大一点

比如8192
